Index: ∞
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><a  class=\"btn btn-primary btn-sm\" id=\"tupianshanchu\" style=\"    position: absolute;   \\n' +\r\n        '    top: 10%;   \\n' +\r\n        '    left: 50%;   \\n' +\r\n        '    width: 100px;' +\r\n        '    height: 30px;' +\r\n        '      margin:-15px 0 0 -50px;\">删除</a>\r\n\r\n wqewqe
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- ∞	(revision 61ad3e34d96c53f37a6463d634b5a87e71ea9231)
+++ ∞	(date 1606124053139)
@@ -3,6 +3,4 @@
         '    left: 50%;   \n' +
         '    width: 100px;' +
         '    height: 30px;' +
-        '      margin:-15px 0 0 -50px;">删除</a>
-
- wqewqe
\ No newline at end of file
+        '      margin:-15px 0 0 -50px;">删除</a>
\ No newline at end of file
